.github/workflows/helm.yaml

```diff
@@ -39,6 +43,28 @@ jobs:
           if [[ -n "$changed" ]]; then
             echo "::set-output name=changed::true"
           fi
+      
+      # Force linting in debug runs with a change (this action only runs on chart and helm.yaml workflow diffs anyways)
+      - name: Change Kind to deployment
+        run: "sed -i \"s|kind: [^ ]*|kind: Deployment|g\" charts/descheduler/values.yaml"
 
       - name: Run chart-testing (lint)
         run: ct lint --config=.github/ci/ct.yaml --validate-maintainers=false
+      
+      # Need a multi node cluster so descheduler runs until evictions
+      - name: Create multi node Kind cluster
+        run: |
+          cat > kind-config.yaml <<EOF
+          # three node (two workers) cluster config
+          kind: Cluster
+          apiVersion: kind.x-k8s.io/v1alpha4
+          nodes:
+          - role: control-plane
+          - role: worker
+          - role: worker
+          EOF
+          kind create cluster --name kind --config kind-config.yaml --wait 2m
+
+      # helm-extra-set-args only available after ct 3.6.0
+      - name: Run chart-testing (install)
+        run: ct install --config=.github/ci/ct.yaml --helm-extra-set-args="--set=kind=Deployment --set=podSecurityPolicy.create=false"
```

Comment:
Would be great to extend the list of steps with running the e2e

Reply:
Makes sense!